ambry-replication/src/test/java/com/github/ambry/replication/ReplicationTest.java

```diff
@@ -171,6 +171,38 @@ public ReplicationTest(short requestVersion, short responseVersion) throws Excep
     replicationConfig = new ReplicationConfig(verifiableProperties);
   }
 
+  /**
+   * Tests replication model is correct obtained from properties
+   * @throws Exception
+   */
+  @Test
+  public void verifyReplicationTypeFromConfig() throws Exception {
```

Comment:
minor: to make the naming consistent, how about renaming this test to `replicationTypeFromConfigTest`?